mmv1/third_party/terraform/website/docs/r/compute_security_policy.html.markdown

```diff
@@ -320,12 +320,24 @@ The following arguments are supported:
 
 * `layer_7_ddos_defense_config` - (Optional) Configuration for [Google Cloud Armor Adaptive Protection Layer 7 DDoS Defense](https://cloud.google.com/armor/docs/adaptive-protection-overview?hl=en). Structure is [documented below](#nested_layer_7_ddos_defense_config).
 
+* `auto_deploy_config` - (Optional) Configuration for [Automatically deploy Adaptive Protection suggested rules](https://cloud.google.com/armor/docs/adaptive-protection-auto-deploy?hl=en). Structure is [documented below](#nested_auto_deploy_config).
```

Comment:
I missed this before, but I think we want to mention that this field is only available in the beta version of the provider. Should be able to replace `(Optional)` with `(Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html))`.